File: src/main/java/com/demonwav/statcraft/commands/CustomResponse.java
/*
 * StatCraft Bukkit Plugin
 *
 * Copyright (c) 2016 Kyle Wood (DemonWav)
 * https://www.demonwav.com
 *
 * MIT License
 */

package com.demonwav.statcraft.commands;

import org.bukkit.command.CommandSender;

public interface CustomResponse {

    void respondToCommand(final CommandSender sender,  String[] args);
}

File: src/main/kotlin/com/demonwav/statcraft/commands/CustomResponse.kt
/*
 * StatCraft Bukkit Plugin
 *
 * Copyright (c) 2016 Kyle Wood (DemonWav)
 * https://www.demonwav.com
 *
 * MIT License
 */

package com.demonwav.statcraft.commands

import org.bukkit.command.CommandSender

interface CustomResponse {

    fun respondToCommand(sender: CommandSender, args: Array<String>)
}
